### PR TITLE
Adds Supabase Apple, Twitter auth support with v0.33-rc

### PR DESCRIPTION
**Pull request by dthyresson**
This PR upgrades the Playground Auth sample app to the v0.33 release candidate that has support for Twitter and Apple logins.

See https://supabase.io/blog/2021/06/02/supabase-beta-may-2021#apple-logins-are-now-available and https://supabase.io/blog/2021/06/02/supabase-beta-may-2021#twitter-logins-are-now-available for Supabase's announcement for Apple and Twitter login support.

Note: Currently neither are configured as Apple requires a dev certification (see https://medium.com/identity-beyond-borders/how-to-configure-sign-in-with-apple-77c61e336003 from more info) and Twitter requires an approved Twitter app.

However, placeholders exist and the redwood auth provider permits these as oauth providers.

There is also a login with refreshToken that Supabase adds likely for use with native apps or clis (ie, not web browsers) but this is not demo'd.

### CODE REVIEW

**Comment by morganmspencer** [comment]
@dthyresson I believe I have an Apple dev cert if we want to implement that. Not sure how we would handle the logistics of that though

**Review by dac09 — APPROVED** [review]
LEft a small comment

File: web/package.json [hunk 1]
```diff
@@ -16,10 +16,10 @@
...
     "@heroicons/react": "^1.0.1",
-    "@redwoodjs/auth": "^0.32.2",
-    "@redwoodjs/forms": "^0.32.2",
-    "@redwoodjs/router": "^0.32.2",
-    "@redwoodjs/web": "^0.32.2",
+    "@redwoodjs/auth": "^0.33.0-rc.16",
+    "@redwoodjs/forms": "^0.33.0-rc.16",
+    "@redwoodjs/router": "^0.33.0-rc.16",
+    "@redwoodjs/web": "^0.33.0-rc.16",
     "@supabase/supabase-js": "^1.11.2",
```
Comment (dac09) [inline web/package.json:23]: Should we bump this?

Comment (dthyresson) [reply web/package.json:23]: Oops. Yes.

**Review by dac09 — APPROVED** [review]

**Pull request merged**